x-pack/plugins/integration_assistant/common/api/model/common_attributes.schema.yaml

```
@@ -36,6 +36,27 @@ components:
       items:
         type: object
 
+    SampleFormat:
```

```suggestion
    SampleLogFormat:
```

For better understanding.

---

But we had `rawSamples`, not `rawLogSamples`

---

Sure , yeah. But I guess the idea of using `SampleFormat` is to identify the `format of samples` and not `a sample format`. So just to make this clear , I had the suggestion. But feel free to choose what you like to use.

---

Perhaps `samplesFormat`? I just want consistency with the existing attributes.

---

Sounds good. Yeah.